Describe Client Throttling behaviour (#57)
appuio/appuio-cloud-docs: docs/modules/ROOT/pages/explanation/client-throttling.adoc

= Client Throttling

> You may see the following similar error message from time to time when interacting with {product} on the command line, especially when using `kubectl` instead of `oc`:

Reviewer: ```suggestion
You may see a similar error message to the one below when interacting with {product} on the command line, especially when using `kubectl` instead of `oc`:
```

----
I1123 16:51:06.257182   77654 request.go:665] Waited for 1.118997154s due to client-side throttling, not priority and fairness, request: GET:https://api.zone.appuio.cloud:6443/apis/apps.openshift.io/v1?timeout=32s
----

> This is described very well in https://access.redhat.com/solutions/5587221[Throttling request took messages reported when running oc commands on OCP 4].

Reviewer: ```suggestion
The cause is described very well in https://access.redhat.com/solutions/5587221[Throttling request took messages reported when running oc commands on OCP 4].
```
